### DISCUSSION TITLE
[Question] Extending CKEditor docs

### DISCUSSION THREAD

**Discussion by khalwat** [craftcms/ckeditor#226] [Q&A] [answered]
Not a bug report as much as a question about whether there are plans to publish information on how third parties can extend the CKEditor implementation in Craft CMS 5 & beyond, so we can add our own plugins, etc.?

**Comment by brandonkelly** [comment, marked as answer]
Our docs for it are in the readme: [Adding CKEditor Plugins](https://github.com/craftcms/ckeditor?tab=readme-ov-file#adding-ckeditor-plugins). There are a few links within there that should help with the CKEditor side of things. It’s a pretty rich (and constantly improving) framework so it would be impractical for us to try to maintain some sort of cliff notes version of its docs.
